plasma_framework/contracts/poc/fast_exits/Liquidity.sol

```diff
@@ -186,6 +188,28 @@ contract Liquidity is ERC721Full {
         }
     }
 
+    /**
+     * @dev Get Exit bond back - to be called by exit intitiator
+     * @param exitId The exit id
+    */
+    function getExitBond(uint160 exitId) public {
+        require(exitData[exitId].bondReturned == false, "Exit Bond Already returned");
+        require(exitData[exitId].exitInitiator != address(0), "ExitId is invalid");
+        require(msg.sender == exitData[exitId].exitInitiator, "Only the Exit Initiator can claim the bond");
+        
+        uint160[] memory exitIdList = new uint160[](1);
+        exitIdList[0] = exitId;
+        PaymentExitDataModel.StandardExit[] memory exits = paymentExitGame.standardExits(
+            exitIdList
+        );
+        if (exits[0].utxoPos == 0) {
```

Comment:
maybe `assert` here. this would eliminate need to use `if else` structure

Reply:
ohh if we use assert here, we cant return back an error message? Do we want that?

Reply:
> require in the middle might not be a good idea?

I don't think it matters. And `if-else + revert` is really just the same thing.

Reply:
ah...I know we are still handling the exit processed check in another issue.
However, can we put `exit.utxoPos == 0` into a function now (and the part in `getWithdraw` too) with naming like: (eg.) `checkExitProcessed` so make this a bit easier to read.

Reply:
>And if-else + revert is really just the same thing

Thats true! I'll change to require for better readability then
Also having a function here to `checkExitProcessed` will make it simpler to make changes later